tests/kubeless-test-client/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM debian:stable-slim
+FROM alpine:latest
 
 LABEL source="git@github.com:kyma-project/kyma.git"
 
-RUN apt-get update && apt-get install -y curl unzip && rm -r /var/lib/apt/lists/*
+RUN apk add --no-cache curl jq
 
-RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl
+RUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
```

Comment:
IMO versions should be explicit in order to keep the tests deterministic

Reply:
Changed.